modules/admin/proc-configuring-postgresql-instance-using-operator.adoc

```diff
@@ -5,13 +5,14 @@ You can configure an external PostgreSQL instance by using the {product} Operato
 
 .Prerequisites
 
+* You are using PostgreSQL version 15.
```

Comment:
Per previous comment, Postgres 14, 15, 16 are supported.  We should just state a supported version of PostgreSQL.